integration/ruler_test.go

```diff
@@ -179,45 +182,106 @@ func TestRulerEvaluationDelay(t *testing.T) {
 	namespace := "ns"
 	user := "fake"
 
+	evaluationDelay := time.Minute * 5
+
 	configOverrides := map[string]string{
 		"-ruler.storage.local.directory":   filepath.Join(e2e.ContainerSharedDir, "ruler_configs"),
 		"-ruler.poll-interval":             "2s",
 		"-ruler.rule-path":                 filepath.Join(e2e.ContainerSharedDir, "rule_tmp/"),
-		"-ruler.evaluation-delay-duration": "5m", // 5 minutes is clarifying when seeing when a rule is evaluated
+		"-ruler.evaluation-delay-duration": evaluationDelay.String(), // 5 minutes is clarifying when seeing when a rule is evaluated
```

Comment:
I'm not sure I understand the comment here

Reply:
Oh I realize you didn't add the comment, sorry.

Reply:
I don't understand it, so it didn't touch it

Reply:
I have removed it now